cli/impl/src/main/java/io/helidon/build/cli/impl/InitCommand.java

```diff
@@ -86,7 +89,24 @@ protected void invoke(CommandContext context) throws Exception {
                 Log.info("Using Helidon version " + helidonVersion);
             } else {
                 String defaultHelidonVersion = defaultHelidonVersion();
-                helidonVersion = prompt("Helidon version", defaultHelidonVersion, this::isSupportedVersion);
+                FileSystem latestArchetype = latestArchetype();
+                ArchetypesData archetypesData = ArchetypesDataLoader.load(latestArchetype);
+                List<String> versions = archetypesData.versions();
+                if (versions.size() > 0) {
+                    int defaultOption = -1;
+                    for (int x = 0; x < versions.size(); x++) {
```

Comment:
We should group the versions (major, major.minor) and limit the number of versions displayed per group.